docs/dashboard_config.md

```diff
@@ -25,15 +28,18 @@ In its default state the Dashboard config is in this form:
 ```yaml
 spec:
   dashboardConfig:
-    disableBYONImageStream: false
-    disableClusterManager: false
-    disableISVBadges: false
-    disableInfo: false
-    disableSupport: false
-    disableTracking: true
-    disableAppLauncher: false
-    disableUserManagement: false
-    enablement: true
+      enablement: true
+      disableInfo: fals
+      disableSupport: false
+      disableClusterManager: false
+      disableTracking: true
+      disableBYONImageStream: false
+      disableISVBadges: false
+      disableAppLauncher: false
+      disableUserManagement: false
+      disableProjects: false
+      disableModelServing: false
+      modelMetricsNamespace: ''
```

Comment:
improper indent -- could you reduce by 2?

Reply:
👍